presto-hive/src/main/java/com/facebook/presto/hive/HiveWriteUtils.java

```diff
@@ -465,7 +465,7 @@ private static void checkWritable(
         verifyOnline(tableName, partitionName, protectMode, parameters);
 
         // verify not read only
-        if (protectMode.readOnly) {
+        if (protectMode.MetadataManager.java) {
```

Comment:
???

Reply:
Ah I fixed it but forgot to push to origin 😂